docs/usage.rst

```diff
@@ -144,9 +144,9 @@ Reading Eclipse data
 
 The ensemble class has specific support for parsing binary files produced
 by reservoir simulator outputting the Eclipse binary format. This support
-is through `libecl_`.
+is through `resdata`_.
```

Comment:
What's that extra underscore?

Reply:
https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

Reply:
Splendid